### DISCUSSION TITLE
Combine stops to unload at 95%

### DISCUSSION THREAD

**Discussion by DaveBrune** [Courseplay/Courseplay_FS22#2546] [Q&A]
Is there a way to make CP wait until the combine is at 100% to unload?  Right now it stops and wants to unload at 95%.  Also, is there a way to make CP extend the offload pipe when it is time to unload?

**Comment by Tensuko** [comment]
If you use AD as well. The pipe will extend always, thats AD overwrite.
No we do 95% max for some reasons and it highly depends on the situation.

**Reply by DaveBrune** [reply]
OK, Thanks.

**Comment by matt-screed** [comment]
Also, if you're using cp to harvest and ad to cart, there is a setting in ad for the combine/harvester that will let it go to 100% before it stops, as well as settings for when it calls a cart (which it does by extending the pipe).  I usually set mine to about 30% because it generally takes until it is almost full for the cart to catch up!
